mp/src/game/server/momentum/mom_player.cpp

```diff
@@ -1573,14 +1573,13 @@ void CMomentumPlayer::TravelSpectateTargets(bool bReverse)
 void CMomentumPlayer::TweenSlowdownPlayer()
 {
     // slowdown when map is finished
-    if (m_Data.m_bMapFinished)
+    if (m_Data.m_bMapFinished && !ConVarRef("mom_mapfinished_movement_enable").GetBool())
```

Comment:
I think it would be better to not call the `TweenSlowdownPlayer` function if the movement_enable var isn't enabled instead, saving you from creating and checking this ConVarRef every tick that the slowdown code would normally happen.

Reply:
Fixed these calls, though not sure where to go to avoid calling this function. Looks like it is a callback function set up when the player spawns. I moved the ConVarRef into the constructor to avoid calls each tick.